beater/beater.go

```diff
@@ -136,10 +114,40 @@ type beater struct {
 	stopped    bool
 }
 
+var once sync.Once
```

Comment:
Sorry, I missed one thing: can this be moved into the Run function, so it's close to where it's used?

```go
var reloadOnce sync.Once
var reloadable = reload.ReloadableFunc(func(ucfg *reload.ConfigWithMeta) error {
    ...
}
```

Reply:
ah yes, didn't know I could do that :+1: